tests/framework/Samplers/VectorSamples/vectorInputModel.py

```diff
@@ -0,0 +1,8 @@
+import numpy as np
```

Comment:
can you add the BEA/RAVEN disclaimer ?

Reply:
We don't have any guidelines for including this in files anywhere, right?  I certainly don't have any feeling for when and on what files it should be included.

Reply:
Yes you are right....we do not have any guideline...and It is worth to mention it somewhere ( I will try to add it in y todolist).
Anyhow, that declaimer is present in all the python files we had when we got opensource (added by Paul Berg).... I guess it does not hurt to do that in any new python file we add... and I will add it in the guideline